packages/react-resizable-panels/package.json

```diff
@@ -9,11 +9,33 @@
     "url": "git+https://github.com/bvaughn/react-resizable-panels.git"
   },
   "source": "src/index.ts",
-  "main": "dist/react-resizable-panels.js",
-  "module": "dist/react-resizable-panels.module.js",
-  "types": "dist/react-resizable-panels.d.ts",
+  "main": "dist/react-resizable-panels.cjs.js",
+  "module": "dist/react-resizable-panels.esm.js",
+  "exports": {
```

Comment:
All of this is a little bit crazy... but well, it's what we found to cover all of the "compatibility boxes". This should work in old nodes, new nodes, with require, with import, with different bundlers, with different versions of TS and its `moduleResolution` setting etc

For example, previously your package was CJS-only and thus in pure-ESM environment it was consumable through a default import (as well as through named ones):
[TS playground](https://www.typescriptlang.org/play?moduleResolution=3&target=99&module=0&ts=5.0.4#code/PTAEAEFsHsBMFcA2BTASsgztR8AuBLaAOwC5Qi5kBGANgCh9IAHaAJ13I1ADNXpJQAclbIAhgGNcAWhEZ8AL1EAjFFKaiiyRBkF0644hg4AFDVtABeTgDpTmxHRChnoAHoB+IA).

This is now fixed - no default export is provided and one can't use a default import to consume this module (that matches how you defined its interface)